pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/codestyle/xml/UnnecessaryLocalBeforeBranch.xml

```diff
@@ -356,4 +356,221 @@ public class Example {
 }
 ]]></code>
     </test-code>
-</test-data>
+    <test-code>
+        <description>Simple unnecessary local before throw</description>
+        <expected-problems>1</expected-problems>
```

Comment:
To be very clear, we should also verify the reported line number. We report the variable declaration, right? (and not the throw statement)

```xml
<expected-linenumbers>3</expected-linenumbers>
```